developer-extension/src/panel/components/tabs/infosTab.tsx

```diff
@@ -35,7 +39,7 @@ export function InfosTab() {
                   'Tracked without Session Replay'
                 )}
               />
-              <Entry name="Created" value={formatDate(Number(infos.cookie.created))} />
+              {infos.cookie.created && <Entry name="Created" value={formatDate(Number(infos.cookie.created))} />}
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** ‏why is this extra check needed?

Reply:
In version 4 `created` was undefined when the session was not tracked which makes `formatDate(Number(infos.cookie.created))` throws and crashes the extension.